web/src/invite.ts

```diff
@@ -292,7 +306,30 @@ function open_invite_user_modal(e: JQuery.ClickEvent<Document, undefined>): void
         user_has_email_set: !settings_data.user_email_not_configured(),
     });
 
+    function hide_invite_user_modal(): void {
+        overlays.close_active_modal();
+    }
+
     function invite_user_modal_post_render(): void {
+        const pill_config = {
+            do_not_show_user_group_size: true,
+        };
+        $pill_container = $(`#invite-user-form .pill-container`).expectOne();
+        pills = user_group_pill.create_pills($pill_container, pill_config);
```

Comment:
Here the type of value returned by the function and type defined for `pills` are incompatible because the function can also return `undefined`. I am not sure what is the best way in such cases.

Reply:
Started https://chat.zulip.org/#narrow/stream/6-frontend/topic/input_pill.20typescript/near/1670357 for this.